playbooks/ceph.yml

```diff
@@ -299,6 +299,14 @@
       vars:
         this_host: "{{ _target_hosts | first }}"
 
+    - name: Assert if any EDPM node's n/w interface is missing in storage network
+      ansible.builtin.assert:
+        that:
+          - cifmw_cephadm_rgw_network is defined
+          - hostvars[item][all_addresses] | ansible.utils.ipaddr(cifmw_cephadm_rgw_network) | length > 0
+        fail_msg: "node {{ item }} doesn't have any interface connected to network {{ cifmw_cephadm_rgw_network }}"
```

Comment:
there are two assertion there - the message "just" talks about the second one, and may very well fail while computing the `fail_msg` in case that `cifmw_cephadm_rgw_network` parameter is indeed undefined..

Thoughts? Can we expect this one to not be defined at that point of the play?

Reply:
Per line 294 we know `cifmw_cephadm_rgw_network` will be defined so I think line 305 can be removed.

Reply:
Yeah it will be always defined at L294, first assertion is removed in the latest patch.